### ISSUE TITLE
vmalert - part of go templates not working in annotations

### ISSUE THREAD

**Issue by ve4eslav** [VictoriaMetrics/VictoriaMetrics#2514]
**Describe the bug**
go template not working properly. In this particular case, "slice" doesn't work, but works in prometheus.

**To Reproduce**
create VMRule
```
  groups:
    - name: monitoring-test-app
      rules:
        - alert: ExapmleAppInstanceDown
          annotations:
            dashboard: >
              [GrafanaDashboard]https://job={{ slice $labels.job 9 }}&var-job={{
              $labels.job }}
            runbook: |
              message for test alert. '{{ printf "%.2f" $value }}%'
              percent: {{ humanizePercentage $value }}
          expr: >-
            sum by(job)(absent(miss_metric{job="vminsert-ret1m-1a",
            namespace="monitoring"}))
          for: 30s
          labels:
            cluster: cluster11
```

**Expected behavior**
The template is expected to contain a label snippet starting at character 10 of label.job (in example: ret1m-1a)

**Logs**
`2022-04-28T11:50:09.770Z	error	VictoriaMetrics/app/vmalert/main.go:293	cannot parse configuration file: errors(1): invalid group "monitoring-test-app" in file "/etc/vmalert/config/vm-apps-rulefiles-0/monitoring-monitoring-test-app.rules.yaml": invalid annotations for rule "monitoring-test-app"."ExapmleAppInstanceDown": errors(1): key "dashboard", template "[GrafanaDashboard]https://job={{ slice $labels.job 9 }}&var-job={{ $labels.job }}\n": error evaluating annotation template: template: :1:99: executing "" at <slice $labels.job 9>: error calling slice: index out of range: 9`
When recreating one of the pods, a fatal error occurs. On the second one only the error above
`2022-04-28T12:13:38.777Z	fatal	VictoriaMetrics/app/vmalert/main.go:132	cannot parse configuration file: errors(1): invalid group "monitoring-test-app" in file "/etc/vmalert/config/vm-apps-rulefiles-0/monitoring-monitoring-test-app.rules.yaml": invalid annotations for rule "monitoring-test-app"."ExapmleAppInstanceDown": errors(1): key "dashboard", template "[GrafanaDashboard]https://job={{ slice $labels.job 9 }}&var-job={{ $labels.job }}\n": error evaluating annotation template: template: :1:99: executing "" at <slice $labels.job 9>: error calling slice: index out of range: 9`

**Version**
VMAlert v1.73.0 and also tested on latest v1.76.1

**Comment by hagen1778** [comment]
Hi @ve4eslav! The fix was merged at https://github.com/VictoriaMetrics/VictoriaMetrics/pull/2528 and will be included to the next release.
Meanwhile, you can try to [build vmalert from sources](https://docs.victoriametrics.com/vmalert.html#how-to-build-from-sources).

**Comment by valyala** [comment]
@ve4eslav , could you verify whether the issue is fixed in [vmalert v1.77.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases)?

**Comment by ve4eslav** [comment]
@hagen1778 , @valyala 
Yes,
now it works as expected.
Thank you.
